GliaWidgets/Sources/ViewController/Common/FilePicker/FilePickerController.Envrionment.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+extension FilePickerController {
+    public struct Environment {
```

Comment:
I guess it can have internal access level

Reply:
fixed [98255c2](https://github.com/salemove/ios-sdk-widgets/pull/979/commits/98255c2f7dfa9effe5b77ec50e8b3c2f5b40a8d5)